Changelog.md

```diff
@@ -4,14 +4,36 @@ This page defines all the changes that were done in the API. There is a differen
 
 ## Changes in Zenya 6.2.0
 
-### Deprecated **icon_base64** properties
-Icons are no longer images and are only a name and color. Therefore the icon_base64 properties on mobile forms and question lists are deprecated. The properties have a static value representing a transparent pixel.
-Affected routes:
-- **GET** `card_files/cards/{card_id:int}/questionlisttemplates`
-- **GET** `/cases/mobile_forms`
-- **GET** `/cases/mobile_forms/{form_id}`
-- **GET** `/card_files/{card_file_id}/mobile_forms` 
-
+Change|Route|Remarks
+|--|--|--|
+Deprecated attribute|**GET** routes returning content_item |Icons are no longer images and are only a name and color. Therefore the `icon_base64` properties on mobile forms and question lists are deprecated. The properties have a static value representing a transparent pixel. Affected routes: GET card_files/cards/{card_id:int}/questionlisttemplates, GET /cases/mobile_forms, GET /cases/mobile_forms/{form_id}, GET /card_files/{card_file_id}/mobile_forms
+Depricated|**GET** `/cases/forms/{form_id}/icon`|Icons are no longer images and are only a name and color
+Depricated|**GET** `/cases/mobile_forms/{form_id}/icon`|Icons are no longer images and are only a name and color
+Added|**GET**,**PATCH**,**DELETE** `/cases/attachments/{attachment_id}`|Manage case attachments by attachment id
+Added|**GET** `/cases/attachments/{attachment_id}/download`|Downloads the attachment
+Added|**GET**,**PATCH**,**DELETE** `/cases/hyperlinks/{hyperlink_id}`|Manage case hyperlinks
+Added|**GET**,**PATCH**,**DELETE** `/cases/notes/{note_id}`|Manage the notes from a case
+Added|**GET**,**PATCH**,**DELETE** `/cases/to_dos/{to_do_id}`|Manage case todo's
+Added|**POST** `/cases/to_dos/{to_do_id}/attachments`|Adds new attachment
+Added|**POST** `/cases/to_dos/{to_do_id}/notes`|Adds new note to a to do
+Added|**GET** `/cases`|Added new filter parameter case_ids
+Changed|**GET** `/cases/case_types`|Added properties to the return type
+Added|**GET** `/cases/case_types/{case_type_id}`|Get case type by case type id
+Added|**GET** `/case_types/{case_type_id}/permissions`|Get the permissions of a case type by case type id
```

Comment:
By case type id hoeft er niet bij , is niets speciaals

Reply:
ok